engine/src/test/scala/ReadTsvWorkflowSpec.scala

```diff
@@ -1,12 +1,9 @@
 package cromwell
 
-import akka.testkit._
-import wdl4s.types.{WdlFileType, WdlIntegerType, WdlStringType, WdlArrayType}
-import wdl4s.values.{WdlFile, WdlInteger, WdlArray, WdlString}
-import cromwell.CromwellSpec.DockerTest
+import akka.testkit.EventFilter
 import cromwell.util.SampleWdl
-
-import scala.language.postfixOps
+import wdl4s.types.{WdlFileType, WdlStringType, WdlIntegerType, WdlArrayType}
+import wdl4s.values.{WdlFile, WdlString, WdlInteger, WdlArray}
```

Comment:
@kshakir @cjllanwarne take a look at where this file was previously (it wasn't in the right spot). The imports were what intellij asked me to do